src/DoubleSlider.elm

```diff
@@ -109,10 +109,26 @@ update message model =
                 newModel =
                     case valueType of
                         LowValue ->
-                            { model | lowValue = convertedValue }
+                            let
+                                newLowValue =
+                                    List.minimum
+                                        [ convertedValue
```

Comment:
Given it's always a comparison of 2, can just use `min : comparable -> comparable -> comparable` here, and avoid the `Maybe` yuckiness